src/adk/command_list.py

```diff
@@ -278,16 +282,18 @@ def applications_upload(
     application_path, application_name = retrieve_application_name_and_path(application_name=application_name)
     validate_dict = processor.applications_validate(application_name=application_name,
                                                     application_path=application_path)
+    validation_messages = format_validation_messages(validate_dict)
+
     if validate_dict["error"] or validate_dict["warning"]:
-        show_validation_messages(validate_dict)
-        typer.echo(f"Application '{application_name}' is invalid. Application not uploaded")
+        typer.echo("Application could not be uploaded.")
```

Comment:
Also here the message is changed a bit. We can but we do not upload invalid apps